src/projectDB/projectversioning.cpp

```diff
@@ -25,7 +25,12 @@ map<Version, int> appDbVersionMap = {
  * format to the next. And DO NOT assume that any of the tables that should be
  * transformed already exist.
  */
-map<int, string> dbVersionUpgradeScripts = {};
+map<int, string> dbVersionUpgradeScripts = {
+    {
+        0, "ALTER TABLE compounds ADD COLUMN note TEXT;"
```

Comment:
Why add `note` column in this branch? Is this branch supposed to be based on some other one?

Reply:
Sorry about this. I was running into an issue with project-versioning when I was working with this and PR #980. Must have staged this change during that mess. Will correct.